src/main/java/com/gmail/goosius/siegewar/command/SiegeWarAdminCommand.java

```diff
@@ -232,10 +234,14 @@ private void parseSiegeWarSiegeCommand(CommandSender sender, String[] args) {
 				Messaging.sendErrorMsg(sender, Translation.of("msg_err_town_not_registered", args[0]));
 				return;
 			}
-			if (!SiegeController.hasActiveSiege(town)) {
+			if (!SiegeController.hasActiveSiege(town) && !args[1].equalsIgnoreCase("setplundered")) {
 				Messaging.sendErrorMsg(sender, Translation.of("msg_err_not_being_sieged", town.getName()));
 				return;
 			}
+			if (!SiegeController.hasSiege(town) && args[1].equalsIgnoreCase("setplundered")) {
+				Messaging.sendErrorMsg(sender, Translation.of("msg_err_not_being_sieged", town.getName()));
+				return;				
+			}

```

Comment:
These two tests nearly duplicate each other, I would think only the 2nd one would be needed, no?

Reply:
They're both needed, it's so that setPlundered can only be used when a siege has finished, and the other commands only able to be used while a siege is still active

Reply:
ok